src/main/resources/templates/fragments/footer.html

```diff
@@ -13,22 +13,68 @@
         </footer>
         <!-- End Footer -->
 
-        <script src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
         <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
                 integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
                 crossorigin="anonymous"></script>
         <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
                 integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
                 crossorigin="anonymous"></script>
-        <script type="text/javascript">
-            function ConfirmDelete() {
-                return confirm("Are you sure you want to delete?");
+<!--        For Delete Item-->
+        <script language="javascript" type="text/javascript">
+            function ConfirmDelete(id) {
+                if (window.confirm('Are you sure you want to delete?')) {
+                    $.ajax({
+                        type : "DELETE",
+                        url : "/delete/"+ id,
+                        success : function() {
+                            console.log("a mers");
```

Comment:
let's keep messages in english

Reply:
this messages was for debugging and will be deleted.